CONTRIBUTING.md

```diff
@@ -1,13 +1,25 @@
 # Contributing guidelines
 
-For anyone who knows [Git](https://git-scm.com) and JavaScript basics: feel free to contribute to this repository! Our code is open source. Be sure to follow our [code of conduct](https://github.com/ScratchAddons/ScratchAddons/blob/master/CODE_OF_CONDUCT.md).
+For anyone who knows JavaScript basics: feel free to contribute to this repository! Our code is open source. Be sure to follow our [code of conduct](https://github.com/ScratchAddons/ScratchAddons/blob/master/CODE_OF_CONDUCT.md).
 
 ## Filing [issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues)
 
-Found a bug? Report it! Be sure to check if it is already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
+Found a bug? Report it in an issue! But first, be sure to check if it was already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
 
 Also, feel free to ask questions, suggest new features, and more.
 
 ## Creating [pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
 
-Contributions are welcome! Be sure to file an issue first (see above). If you know a fix for that issue, or you know a fix for a different one, you can create a pull request. Fork this repository, create a new branch from the 'master' branch, and make your changes on the new branch. Now, create a pull request on the origin repository (ScratchAddons/ScratchAddons). We will review your pull request.
+Contributions are welcome! If you have something in mind, it's best if you file an issue first (see above). That way, we can discuss it before you start a pull request. If you know a fix for that issue (or any other issue), you can follow the steps below to create a pull request.
+
+If you're making a new addon, our [FAQ page](https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/) will teach you how to make one.
+
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) this repository if you haven't already.
+2. [Sync your fork's `master` branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) to ensure it is up to date.
+3. [Clone Scratch Addons](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository) so you can make and test your changes.
+4. [Create a new branch](https://github.com/firstcontributions/first-contributions/blob/main/README.md#create-a-branch) from the `master` branch.
+5. [Create, commit, and push](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-and-pushing-changes) the changes you want to make. Make sure you've selected your new branch.
```

Comment:
These steps aren't quite as clear as they could be, but it was a bit hard to put them together when links are involved and I've never used Git. I think this is good enough, what do you say?